### PR TITLE
MNGSITE-398 Fix typos and make improvements in `Configuring Apache Maven`

### PR DESCRIPTION
**Pull request by EPadronU**

### CODE REVIEW

File: content/markdown/configure.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ configuration for Maven usage across projects.
 
 ## `.mvn` folder:
 
-Located with in the projects top level folder, the files `maven.config`, `jvm.config`, and `extensions.xml`
+Located within the projects top level folder, the files `maven.config`, `jvm.config`, and `extensions.xml`
```
Comment (michael-o) [inline content/markdown/configure.md]: make it `directory`

File: content/markdown/configure.md [hunk 2]
```diff
@@ -78,8 +78,8 @@ Starting with Maven 3.3.1+ you can define JVM configuration via `${maven.project
 
         -Xmx2048m -Xms1024m -XX:MaxPermSize=512m -Djava.awt.headless=true
 
-You don’t need to remember of using this options in `MAVEN_OPTS` or switching between different configurations.
-in the end, add the following:
+You don’t need to use these options in `MAVEN_OPTS`, nor switching between
```
Comment (michael-o) [inline content/markdown/configure.md]: Funny thing `don’t` does not contain an apostrophe. It is in UTF-8 `0xe28099`.

**Comment by michael-o** [comment]
Looks good, please squash.

**Comment by EPadronU** [comment]
Done.

File: content/markdown/configure.md [hunk 3]
```diff
@@ -33,7 +33,7 @@ configuration for Maven usage across projects.
 
 ## `.mvn` folder:
 
-Located with in the projects top level folder, the files `maven.config`, `jvm.config`, and `extensions.xml`
+Located within the projects top level directory, the files `maven.config`, `jvm.config`, and `extensions.xml`
```
Comment (elharo) [inline content/markdown/configure.md]: project's

File: content/markdown/configure.md [hunk 4]
```diff
@@ -78,8 +78,8 @@ Starting with Maven 3.3.1+ you can define JVM configuration via `${maven.project
 
         -Xmx2048m -Xms1024m -XX:MaxPermSize=512m -Djava.awt.headless=true
 
-You don’t need to remember of using this options in `MAVEN_OPTS` or switching between different configurations.
-in the end, add the following:
+You don't need to use these options in `MAVEN_OPTS`, nor switching between
```
Comment (elharo) [inline content/markdown/configure.md]: ", nor switching" -->or switch

File: content/markdown/configure.md [hunk 5]
```diff
@@ -22,8 +22,8 @@ in a number of places:
 
 ## `MAVEN_OPTS` environment variable:
 
-This variable contains parameters used to start up the JVM running Maven and can 
-be used to supply additional options to globally to Maven. E.g. JVM memory 
+This variable contains parameters used to start up the JVM running Maven and
+can be used to globally supply additional options to it. E.g. JVM memory
```
Comment (elharo) [inline content/markdown/configure.md]: consider deleting "globally". It's not needed.

File: content/markdown/configure.md [hunk 6]
```diff
@@ -78,8 +78,7 @@ Starting with Maven 3.3.1+ you can define JVM configuration via `${maven.project
 
         -Xmx2048m -Xms1024m -XX:MaxPermSize=512m -Djava.awt.headless=true
 
-You don’t need to remember of using this options in `MAVEN_OPTS` or switching between different configurations.
-in the end, add the following:
+You don't need to use these options in `MAVEN_OPTS`, or switch between different configurations.
```
Comment (elharo) [inline content/markdown/configure.md]: no comma before "or"

**Pull request merged**